plugin/storage/integration/elasticsearch_test.go

```diff
@@ -100,9 +96,10 @@ func (s *ESStorageIntegration) initializeES(t *testing.T, allTagsAsFields bool)
 		DiscoverNodesOnStart: false,
 	})
 	require.NoError(t, err)
+	s.bulkProcessor, err = s.client.BulkProcessor().BulkActions(1).FlushInterval(time.Nanosecond).Do(context.Background())
```

Comment:
I don't believe we need bulk processor. First, it's not actually the one that is being used to write the data (which is encapsulated in the Client). Second, I just tried an experiment (#5357) of making esRefresh function no-op, and all the tests passed just fine. I suggest removing esRefresh and removing all mentioning of bulkProcessor in this test.

Reply:
9f801dd0d23acbc4ced4111cf9b697769838df73